scripts/test_S1_performance.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+function join_by {
+  local d=${1-} f=${2-}
+  if shift 2; then
+    printf %s "$f" "${@/#/$d}"
+  fi
+}
+
+function test_S1 {
+    # Edit the config file to point to a single S1 option, e.g.,
+    echo 'CVMFS_SERVER_URL="'"$1"'"' | sudo tee /etc/cvmfs/domain.d/eessi-hpc.org.local > /dev/null
```

Comment:
If someone has a private stratum 1 host listed in the /etc/cvmfs/domain.d/eessi-hpc.org.local file, this would be overwritten. Perhaps that should be temporarily saved somewhere to be restored after the benchmark?

Another remark: if someone interrupts this scripts then the preferred S1 would be the one currently in CVMFS_SERVER_URL

Reply:
I think both of these could be implemented with a cleanup function:
```
function cleanup()
{
    # ...
}

trap cleanup EXIT
```

Reply:
Included in https://github.com/EESSI/eessi-demo/pull/24/commits/29286eda273cc3b9c7add71b0c06526ab32c43f2